instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/internal/JdbcPreparedStatementStringifier.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jdbc.internal;
+
+import java.net.URL;
+import java.sql.Date;
+import java.sql.RowId;
+import java.sql.Time;
+import java.sql.Timestamp;
+
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public class JdbcPreparedStatementStringifier {
+
+  private JdbcPreparedStatementStringifier() {}
+
+  public static String stringifyParameter(String value) {
+    return String.format("'%s'", value);
+  }
+
+  public static String stringifyParameter(Number value) {
+    return String.format("%s", value);
+  }
+
+  public static String stringifyParameter(boolean value) {
+    return String.format("%s", value);
+  }
+
+  @SuppressWarnings("DefaultLocale")
+  public static String stringifyParameter(byte value) {
+    return String.format("0x%02x", value);
```

Comment:
this could be misleading, byte is a signed value while the hex representation is unsigned